### ISSUE TITLE
trans add null suffix to non-lating source strings to tranlsate

### ISSUE THREAD

**Issue by MurzNN** [soimort/translate-shell#223]
`trans` script adds `null` suffix to input string at end of line.  Reproducible with fresh 0.9.6.7-release and older versions.
Example:
```
$ trans 'привет'
привет null
(null privet)

Hi null
/null null/

Definitions of привет null
[ Русский -> English ]

noun
    regards
        привет, поклон
    regard
        внимание, отношение, уважение, взгляд, взор, привет
    remembrances
        привет
    compliment
        комплимент, похвала, любезность, поздравление, привет, поклон
    remembrance
        память, воспоминание, сувенир, привет, подарок на память
```
If I pass english string - null is not added:
```
$ ./trans 'hello'
hello
/heˈlō,həˈlō/

Здравствуйте
(Zdravstvuyte)

hello – определения
[ English -> Русский ]

глагол
    здороваться
        greet, hello, salute, hallo, hullo, bow
    звать
        call, invite, shout, hail, hallo, hello
    окликать
        hail, holler, call, challenge, speak, hello

имя существительное
    приветствие
        greeting, welcome, salute, salutation, hello, hail
    приветственный возглас
        hallo, halloa, hello, viva
    возглас удивления
        hallo, halloa, hello

hello
    Здравствуйте, Привет
```

**Comment by soimort** [comment]
It seems that your `trans` and `./trans` are in two different paths. You can verify this via:

```
$ trans -V
```

```
$ ./trans -V
```

Please delete the outdated version.

**Comment by MurzNN** [comment]
Yes, `trans` is older **0.9.5** version, so seems this fixed in **0.9.6.7-release**
